test/unittest/layers/layers_standalone_common_tests.cpp

```diff
@@ -84,7 +84,7 @@ TEST_P(LayerSemantics, gettersValidate_p) {
 TEST_P(LayerSemantics, setBatchValidate_p) {
   nntrainer::TensorDim in_dim({1, 1, 1, 1});
   nntrainer::InitLayerContext init_context =
-    nntrainer::InitLayerContext({in_dim}, 1);
+    nntrainer::InitLayerContext({in_dim}, 1, "layer");
```

Comment:
just an idea, model file might want to check for those kind of invariants: every tensor name has to be unique

Reply:
When requesting weights, each weight name is ensured to be unique by the manager for all its tensor. Its added in #1487.